### PR TITLE
Improved command handling and added resource filtering

### PR DESCRIPTION
**Pull request by Awesome1221**

### CODE REVIEW

File: src/main/java/com/violeth/blockjs/blockjs/CommandInterface.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            for(var i = 0; i < Commands.list.size(); i++) {
+                var commandInst = Commands.list.get(i);
+                if(args[0].equalsIgnoreCase(commandInst.key)) {
+                    commandInst.handle(sender, args);
+
+                    return true;
+                }
+            }
+
+            sender.sendMessage(BlockJS.getPluginMessagePrefix().append(
```
Comment (TheGreatViolet) [inline src/main/java/com/violeth/blockjs/blockjs/CommandInterface.java:25]: I'm not entirely sure what the `BlockJS.getPluginMessagePrefix()` was meant to be, but no such method exists on the BlockJS class.

File: src/main/java/com/violeth/blockjs/blockjs/commands/VersionCommand.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+public class VersionCommand extends Command {
+    {
+        key = "version";
+    }
+    @Override
+    public void handle(CommandSender sender, String[] args) {
+        sender.sendMessage(BlockJS.getPluginMessagePrefix().append(
+            Component.text("The current version: ").append(
+                Component.text(BlockJS.instance.getDescription().getVersion()).color(TextColor.color(0xDDDDDD)).decorate(TextDecoration.ITALIC)
```
Comment (TheGreatViolet) [inline src/main/java/com/violeth/blockjs/blockjs/commands/VersionCommand.java:18]: There also isn't an `instance` field on the BlockJS class

**Review by TheGreatViolet — CHANGES_REQUESTED** [review]
Also one more thing, the import for the CommandInterface class wasn't updated in the main BlockJS class.

Pull request looks solid, just fix those few things and I'd be happy to merge

File: src/main/java/com/violeth/blockjs/blockjs/commands/VersionCommand.java (hunk 2 above)
Comment (TheGreatViolet) [reply src/main/java/com/violeth/blockjs/blockjs/commands/VersionCommand.java:18]: I'm assuming what you were doing was trying to get the version, which in that case you can use something like `Bukkit.getServer().getPluginManager().getPlugin("BlockJS").getDescription().getVersion()` to get the version

Comment (Awesome1221) [reply src/main/java/com/violeth/blockjs/blockjs/commands/VersionCommand.java:18]: I have no clue what happened to the commited file, but here it is

**Review by TheGreatViolet — APPROVED** [review]

**Comment by TheGreatViolet** [comment]
Alright, looks good to me! Going to merge, thanks for the contribution!

**Pull request merged**